### DISCUSSION TITLE
How to use ts-sdk to generate a multisig account and submit transaction with single call

### DISCUSSION THREAD

**Discussion by alex4506** [aptos-labs/aptos-developer-discussions#72] [Questions] [answered]
Need a ts-sdk demo, which is include:
1. Generate a multisig account
2. Submit transaction with single call after collecting needed signatures
3. Add / Remove manager from / to the multisig account

**Comment by 0xmaayan** [comment, marked as answer]
Hi, you can follow this multisig example https://github.com/aptos-labs/aptos-ts-sdk/blob/main/examples/typescript-esm/multisig_v2.ts
